src/main/resources/auth-delegate_messages.xml

```diff
@@ -15,6 +15,22 @@
   <entry key="CAD-007">ID returned by authorization lookup doesn't match requested: {} != {}</entry>
   <entry key="CAD-008">Cookie found, but it didn't grant access: {}</entry>
   <entry key="CAD-009">The code to indicate whether access is allowed was not expected: {} [{}]</entry>
-  <entry key="CAD-010"></entry><!-- This can be used for a new message -->
+  <entry key="CAD-010">Pre-authorized access granted: open access</entry>
+  <entry key="CAD-011">Pre-authorized access requested: tiered</entry>
+  <entry key="CAD-012">Pre-authorized access requested: all-or-nothing</entry>
```

Comment:
The messages are intended also to give insight into which part of the code (the delegate) is active (vs. just informing about the process). I added them because I was having trouble tracking where a problem was happening and this allowed me to trace through the steps being performed.

Reply:
If we want to provide logging that's more geared towards what's being done, I'd suggest an INFO. I don't know that that's needed though?

Reply:
Ok, fair enough, disregard 👍🏻